druid/src/widget/widget_ext.rs

```diff
@@ -178,12 +178,22 @@ pub trait WidgetExt<T: Data>: Widget<T> + Sized + 'static {
     /// Wrap this widget in a [`LensWrap`] widget for the provided [`Lens`].
     ///
     ///
-    /// [`LensWrap`]: ../struct.LensWrap.html
-    /// [`Lens`]: ../trait.Lens.html
+    /// [`LensWrap`]: ../lens/struct.LensWrap.html
+    /// [`Lens`]: ../lens/trait.Lens.html
     fn lens<S: Data, L: Lens<S, T>>(self, lens: L) -> LensWrap<T, L, Self> {
         LensWrap::new(self, lens)
     }
 
+    /// Wrap a `Widget<()>` in a [`UnitWrap`] widget.
+    ///
+    /// [`UnitWrap`]: ../lens/struct.UnitWrap.html
+    fn unit(self) -> UnitWrap<Self>
```

Comment:
I'm starting to have some concerns about how general the names we're using in `WidgetExt` is. I'd be happy to hear some opinions.

I like `with_` for things that take an argument; I could imagine `lens` being `with_lens`. I'm not sure what makes sense here? `into_`?

One option: if we think of `UnitWrap` as an escape hatch, we could exclude it from `WidgetExt`.